src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -74,7 +76,7 @@ public ProcessMetrics(ProcessInstrumentationOptions options)
             unit: "s",
             description: "Total CPU seconds broken down by different states.");
 
-        this.meterInstance.CreateObservableGauge(
+        MeterInstance.CreateObservableGauge(
             "process.cpu.utilization",
```

Comment:
If I recall, the `process.cpu.utilization` metric is the main one that's complicating things, correct? This is due to the state that the instrument would need to maintain.

At this point in time, I think we need to remove it from this instrumentation. I suggest we consider following what java has done and define our own conventions https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/runtime-environment-metrics.md#jvm-metrics.

Reply:
@Yun-Ting Did a specific customer need `process.cpu.utilization`? If not let's remove it and fight this battle if/when someone asks for it 😄

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/972